INSTALL.make.md

```diff
@@ -0,0 +1,257 @@
+QUICK INSTALLATION PROCEDURE.
+=============================
+
+1. ./configure
+2. make
+3. make install (you may need superuser rights)
+
+INSTALLATION PROCEDURE IN DETAILS (NORMAL USERS).
+=================================================
+
+1- Check that you have the OCaml compiler installed on your
```

Comment:
Is the use of dashes instead of dots on purpose? I guess it is, to control the numbering. But the rendering is pretty bad. To use dots, you will just need to indent the part on configure options.

Reply:
Done.